internal/buildscripts/packaging/release/helpers/release_args.py

```diff
@@ -18,46 +18,17 @@
 from .constants import (
     ARTIFACTORY_URL,
     DEFAULT_ARTIFACTORY_USERNAME,
-    DEFAULT_STAGING_USERNAME,
     DEFAULT_TIMEOUT,
     EXTENSIONS,
-    S3_BUCKET,
     STAGES,
-    STAGING_URL,
 )
 
 from .util import Asset
 
-def add_staging_args(parser):
-    signing_args = parser.add_argument_group("Signing Credentials")
-
-    signing_args.add_argument(
-        "--staging-user",
-        type=str,
-        default=os.environ.get("STAGING_USERNAME", DEFAULT_STAGING_USERNAME),
-        metavar="STAGING_USERNAME",
-        required=False,
-        help=f"""
-            Staging username for {STAGING_URL}.
-            Defaults to the STAGING_USERNAME env var if set, otherwise '{DEFAULT_STAGING_USERNAME}'.
-        """,
-    )
-    signing_args.add_argument(
-        "--staging-token",
-        type=str,
-        default=os.environ.get("STAGING_TOKEN"),
-        metavar="STAGING_TOKEN",
-        required=False,
-        help=f"""
-            Staging token for {STAGING_URL}.
-            Required if the STAGING_TOKEN env var is not set.
-        """,
-    )
-
 
 def check_signing_args(args):
-    assert args.staging_user, f"Staging username not set for {STAGING_URL}"
-    assert args.staging_token, f"Staging token not set for {STAGING_URL}"
+    assert args.staging_user, f"Staging username not set"
+    assert args.staging_token, f"Staging token not set"
```

Comment:
I think we can just remove this function since it's not used anymore.

Reply:
I wasn't sure, there are references to it and I tried to just remove references to the repo. If you don't mind, since folks have approved this PR, we can merge it in as is and come back to remove more down the road.